### DISCUSSION TITLE
q-tabs instantaneous switch

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#12008] [General - Components / Directives / etc] [answered]
Hello,

is it possible to make the tabs switch instantaneously? E.g. without the "slide-in" effect.

**Comment by pikil** [comment, marked as answer]
Is that happening while you still have `animated` in `<q-tab-panels />`?

**Reply by i-mage** [reply]
Awesome, that did the trick. Somehow missed it in the documentation. Thanks!
